htdocs/xoops_lib/Xoops/Core/Assets.php

```diff
@@ -96,7 +96,7 @@ class Assets
      * @var string string to identify Assetic filters using instanceof
      */
 
-    private $filterInterface = '\Assetic\Filter\FilterInterface';
+    private $filterInterface = '\\Assetic\\Filter\\FilterInterface';
```

Comment:
> To specify a literal single quote, escape it with a backslash (\). To specify a literal backslash, double it (\\). All other instances of backslash will be treated as a literal backslash: this means that the other escape sequences you might be used to, such as \r or \n, will be output literally as specified rather than having any special meaning.

Parsing this text into concise rules gives this:

- The sequence \' always results in a single quote
- The sequence \\ always results in one backslash character
- In any other circumstance, a \ has no special meaning, and results in one backslash character

So, in the context of a string representing a fully qualified namespaced class name enclosed in single quotes, using double backslashes is neither required or recommended. It will work as double, and it will work as single.

Inside single quotes, for consistency and readability, our standard should be using single backslashes unless there is a compelling requirement otherwise.